R/human_infection.R

```diff
@@ -540,9 +603,29 @@ schedule_infections <- function(
         'A',
         variables$infectivity,
         parameters$ca,
-        variables$progression_rates,
+        variables$recovery_rates,
```

Comment:
Judging from [above](https://github.com/mrc-ide/malariasimulation/pull/320/files#diff-12d8549a471c17bfc3f717bb09830ff339757ca9a0362eb56e41ee47b346a9c1L521), "progression_rates" hasn't been renamed to "recovery_rates" in this version